mozilla/bedrock
# Fix Bug 1357373 - Remove Aurora from Android pre-release channel page

## Description

* Remove Aurora from the [Android channel page](https://www.mozilla.org/en-US/firefox/channel/android/)
* Redirect the [Android Aurora /all/ download page](https://www.mozilla.org/en-US/firefox/android/aurora/all/) to the [Android Nightly /all/ download page](https://www.mozilla.org/en-US/firefox/android/nightly/all/)

## Bugzilla link

[Bug 1357373](https://bugzilla.mozilla.org/show_bug.cgi?id=1357373)

## Testing

* Open the Android channel page and make sure Aurora is gone.
* Open the Aurora /all/ page and make sure it's redirected to the Nightly /all/ page

## Checklist

Cc @flodolo for the removal of strings.
